.infrastructure/pv.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+ name: pv-data
+ namespace: todoapp
+spec:
+ storageClassName: standard
+ persistentVolumeReclaimPolicy: Delete
+ accessModes:
+  - ReadWriteMany
+ capacity:
+  storage: 1Gi
+ hostPath:
+  path: /data/
```

Comment:
Using `hostPath` for PersistentVolumes is generally not recommended for production environments, particularly in multi-node clusters, as it ties the volume to a specific node. Consider using a more robust storage solution like NFS or a cloud provider's persistent storage.